lib/ddtrace.rb

```diff
@@ -35,6 +36,7 @@ module Datadog
   extend Contrib::AutoInstrument
   # Load Contrib extension to global Datadog objects
   Configuration::Settings.include Contrib::Extensions::Configuration::Settings
+  Configuration::TracingValidationProxy.include Contrib::Extensions::Configuration::ValidationProxy
```

Comment:
I suggest moving this to `validation_proxy.rb`, so that the whole module can be examined in that file (otherwise it can get a bit confusing that additional behavior is added elsewhere).